### PR TITLE
fix(#1574): '@angular/http' module not exist

### PR DESCRIPTION
**Pull request by DAB0mB**

### CODE REVIEW

**Comment by DAB0mB** [comment]
I signed it!

File: package.json [hunk 1]
```diff
@@ -37,6 +36,9 @@
     "systemjs": "0.19.38",
     "zone.js": "^0.6.23"
   },
+  "peerDependencies": {
```
Comment (jelbourn) [inline package.json]: This isn't the `package.json` that's used to publish. That one is in `src/lib`

Comment (jelbourn) [inline package.json]: Strike that last idea- the import paths would still be a problem.

**Comment by DAB0mB** [comment]
@jelbourn whether there's a better solution or not as a temporary fix it's better than what's already published. I will make that fix you mentioned above and I suggest in the near future some of you will come up with a better solution.

**Comment by jelbourn** [comment]
LGTM, thanks!

**Pull request merged**